src/main/java/seedu/address/logic/parser/AddressBookParser.java

```diff
@@ -50,20 +57,32 @@ public Command parseCommand(String userInput) throws ParseException {
         case AddCommand.COMMAND_WORD:
             return new AddCommandParser().parse(arguments);
 
+        case MeetCommand.COMMAND_WORD:
```

Comment:
I think this list was initially in alphabetical order, can we continue following that? Or an alternative is to have all `addressBook` commands first and then all the `meetingBook` commands.

Reply:
I don't think it was in alphabetical order initially because ListCommand was before the HelpCommand. I just placed the commands in their little sections etc group all add related commands together for meetings and person